packages/studioCMS/src/components/exports/CImage.astro

```diff
@@ -3,7 +3,7 @@ import { Image } from 'astro:assets';
 import Config from 'virtual:studiocms/config';
```

Comment:
when it finally moves to the ImageHandler integration.... lol i've been slowly working twords ripping the `CustomImage` component into the actual Internal imagehandler integration instead of coming from the main integrations exports.  I will note the only reason its called CImage instead of image, is because of the AstroLSP complaining when i called it Image and used the astro Image component on the inside... :sweat_smile:

Reply:
Could it be renamed to `CustomImage.astro` for now?

Reply:
done in [7a17eef](https://github.com/astrolicious/studiocms/pull/77/commits/7a17eefe47128c104d28000879462bbf134c12a2)